### PR TITLE
Add back nicer activity line

### PR DESCRIPTION
**Pull request by estufar**
This adds back in the (slightly) prettier activity line that was removed in #977 due to a bug in Chrome that caused the chat menu to disappear. I found a workaround that keeps both the semitransparent white activity line and the chat menu. It's slightly hacky, but as far as I can tell the workaround doesn't cause any issues with the rest of XKit or Tumblr, or with Firefox.

Also blacklisted another page I missed last time.

### CODE REVIEW

File: Extensions/estufars_sidebar_fix.css [hunk 1]
```diff
@@ -141,8 +141,22 @@
...
+   work around this by setting right column z-index which makes it appear again */
+
+.right_column {
+	z-index: 1;
+}
+
+/* z-index change will mess with the first section
+   (another chrome issue, will be fixed in chrome 50) */
+
+.right_column .popover_subsection:first-child .popover_header, .right_column .popover_subsection:first-child .popover_header>.popover_item_suffix {
```
Comment (finagle29) [inline Extensions/estufars_sidebar_fix.css]: prefer newline after comma

**Comment by finagle29** [comment]
this has some visibility issues with the darkest themes, is there an easy way to set the color of the activity line to be `rbga(255,255,255,0.5)` for example? if not, that can be something for themes developers to look out for and fix as needed

**Comment by estufar** [comment]
Since Tumblr's drawing the line on a canvas there's not much that we can do about the color besides this. If a theme sets the background of `#right_column` or `.l-content` to an opaque solid, the css blend mode will make it essentially `rbga(255,255,255,0.5)`. However, if the right column has a transparent background to let an image show through then the blending won't work and it will fall back to the grey color that it was originally.

Even more irritating, it can be made so that it's consistently semitransparent white but there's some weird bug in Chrome such that if you do that it causes the problem in #977

**Comment by hobinjk** [comment]
LGTM
@homu r+

**Pull request merged**